snippets/deepFreeze.md

```diff
@@ -0,0 +1,38 @@
+### deepFreeze
+
+Deep freezes an object.
+
+Calls `Object.freeze(obj)` recursively on all unfrozen properties of obj that are `typeof` function or object.
+
+```js
+const deepFreeze = obj => {
+  Object.freeze(obj);
```

Comment:
This can be moved into the return statement, as the passed `obj` will be returned from `Object.freeze()`.